src/Glimpse.Client.Web/SpaClientResource.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Glimpse.Server.Web;
+using Microsoft.AspNet.Http;
+using Microsoft.Net.Http.Headers;
+
+namespace Glimpse.Client.Web
+{
+    //TODO: "Merge" with ClientResource
+    public class SpaClientResource : IResource
+    {
+        public async Task Invoke(HttpContext context, IDictionary<string, string> parameters)
+        {
+            var response = context.Response;
+            response.Headers[HeaderNames.ContentType] = "text/html";
+
+            await response.WriteAsync(@"
+<html>
+<head><meta http-equiv='refresh' content='3; url=/glimpse/client/index.html' ></head>
+<body><h1>We must reconcile this with <a href='/glimpse/client/index.html'>ClientResource</a>.</h1><p>Auto redirect in 3 seconds....</p></body>
+</html>");
+        }
+
+        public string Name => "SpaClientScript";
+        public ResourceParameters Parameters => new ResourceParameters(+ResourceParameter.Hash, ResourceParameter.RequestId);
```

Comment:
Sure, it's neat, but it's wasn't exactly intuitive. Does it even show up in IntelliSense? I think I'd go for something like `ResourceParameter.Hash(isRequired: true)`.